bin/server.js

```diff
@@ -73,7 +74,8 @@ var launch = function(profile) {
       context: {
         validator:      validator,
         Client:         Client,
-        azureAccounts:  JSON.parse(cfg.get('auth:azureAccounts'))
+        azureAccounts:  JSON.parse(cfg.get('auth:azureAccounts')),
+        sts:            new AWS.STS(cfg.get('aws'))
```

Comment:
AWS STS credentials works globally: http://docs.aws.amazon.com/STS/latest/UsingSTS/Welcome.html

So that should be fine. Note at this point this is only for S3 access. Special case access like EC2 should be done with custom IAM users and custom policies.